### DISCUSSION TITLE
[Art competition] First-order logic

### DISCUSSION THREAD

**Discussion by Gimongjin** [prosyslab-classroom/cs424-program-reasoning#257] [Art competition]
![DALL·E 2022-11-30 21 16 02 - Very huge and beautiful sand sculpture ](https://user-images.githubusercontent.com/79095967/204799337-73f48e4a-eee3-4df4-ba39-5acfa1439e76.png)

## Short Description
For me, First-order logic means very huge and beautiful sand sculpture.

## Long Description (up to 3 sentences)
First order logic is a formal system that are built by very small, simple pieces of quantifiers, terms and predicates like sand. But it can describe various logical sentences from simple and small ones to complicated and large ones. Also they are very beautiful like well-made sand sculptures.

**Comment by KihongHeo** [comment]
Good. Then what would be (un)decidability of FOL?
